src/test/rustdoc/doc-auto-cfg.rs

```diff
@@ -1,14 +1,40 @@
 #![feature(doc_auto_cfg)]
-
 #![crate_name = "foo"]
 
 // @has foo/fn.foo.html
-// @has - '//*[@class="item-info"]/*[@class="stab portability"]' 'non-doctest'
-#[cfg(not(doctest))]
+// @has - '//*[@class="item-info"]/*[@class="stab portability"]' 'non-meowmeow'
+// compile-flags:--cfg feature="meowmeow"
```

Comment:
No need to put this one multiple times. Also, please put it at the top of the file.

Reply:
Oh, turns out we don't need it at all, `meowmoew` is a *cfg* and not a *feature*. I'm going to delete it.